frontend/src/toolbar/utils.ts

```diff
@@ -69,7 +69,7 @@ export function elementToSelector(element: ElementType): string {
             .map((a) => `.${cssEscape(a)}`)
             .join('')
     }
-    if (element.href) {
+    if (element.href && element.tag_name === 'a') {
```

Comment:
[href=*] selectors only work on anchor elements?

Reply:
The `href` attribute just doesn't exist on a `button`. Adding it in the selector always results in 0 found elements.

[Theoretically](https://www.w3schools.com/tags/att_href.asp), it also exists on `area`, `base` and `link` elements... but... 1) base and link tags are invisible and 2) nobody uses image maps (for `area`).... though we could support them I suppose? 🤔

Reply:
Ah, so the the parent anchor's `href` was pulled into its child's selector 👍  (well, technically a button could have `href` too, but only for some dev purposes really).